ui.tests/test-module/specs/replaceaction/replacelayout-sites.authoring.spec.js

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright 2023 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+const sitesSelectors = require('../../libs/commons/sitesSelectors'),
+    afConstants = require('../../libs/commons/formsConstants');
+
+/**
+ * Testing Form Button with Sites Editor
+ */
+describe('Button - Authoring', function () {
+    // we can use these values to log in
+
+    const dropButtonInSites = function () {
+        const dataPath = "/content/core-components-examples/library/adaptive-form/button/jcr:content/root/responsivegrid/demo/component/guideContainer/*",
+            responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+        cy.selectLayer("Edit");
+        cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Button", afConstants.components.forms.resourceType.formbutton);
+        cy.get('body').click(0, 0);
+    }
+
+    const testButtonBehaviour = function (buttonEditPathSelector, buttonDrop) {
+        var resetButton = "[value='/apps/forms-components-examples/components/form/actions/reset']",
+            submitButton = "[value='/apps/forms-components-examples/components/form/actions/submit']";
+
+        dropButtonInSites();
+        cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + buttonEditPathSelector);
+        cy.invokeEditableAction("[data-action='replace']"); // this line is causing frame busting which is causing cypress to fail
+        // Check If Dialog Options Are Visible
+        cy.get(resetButton)
+            .should("exist");
+        cy.get(submitButton)
+            .should("exist");
+        cy.get(resetButton)
+            .click();
+
+        // Checking some dynamic behaviours
```

Comment:
pls also check in the editor if button editable is removed (not present) and reset editable is present instead in the content frame.